frontend/lib/laletterbuilder/letter-builder/habitability/tests/issues.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { preloadLingui } from "../../../../tests/lingui-preloader";
+import { LaLetterBuilderLinguiI18n } from "../../../site";
+import { AppTesterPal } from "../../../../tests/app-tester-pal";
+import React from "react";
+import { Route } from "react-router-dom";
+import { newSb } from "../../../../tests/session-builder";
+import { LaLetterBuilderRouteComponent } from "../../../routes";
+import { LaLetterBuilderRouteInfo } from "../../../route-info";
+
+beforeAll(preloadLingui(LaLetterBuilderLinguiI18n));
+
+const sb = newSb().withLoggedInJustfixUser();
+
+describe("issues page", () => {
```

Comment:
should this also test that the error message shows up when the user attempts to submit without an issue?